### ISSUE TITLE
Block Mixing Erosion Brush

### ISSUE THREAD

**Issue by sw3aterCS** [14ercooper/14erEdit#113]
**Is your feature request related to a problem? Please describe.**
When I have two different block var palettes in an area, it's always a hassle trying to transition the two.

**Describe the solution you'd like**
An erosion brush that would shuffle the blocks in the area it is applied would make transitioning the borders between different blockvar palettes easier.

**Describe alternatives you've considered**
I *can* do this manually by hand by taking whatever operators that created both blockvars and merging them into one brush command with a 50% odds for one and a 50% odds for the other. However, typing it all out is a bit of a hassle.

**Additional context**
Basically something that can turn this
![2021-01-16_11 44 33](https://user-images.githubusercontent.com/29156621/104821458-e030e380-57f0-11eb-89b5-8522608ce5b9.png)
into this
![2021-01-16_11 45 17](https://user-images.githubusercontent.com/29156621/104821465-e8891e80-57f0-11eb-95c3-60755182a140.png)

**Comment by 14ercooper** [comment]
You do know it'll clsoe itself when I merge that branch into master, right?
